### DISCUSSION TITLE
Critical Alert - Check Battery?

### DISCUSSION THREAD

**Discussion by jcleek** [jgyates/genmon#1137] [General]
![Voltage](https://github.com/jgyates/genmon/assets/16121417/d3611ab6-77e2-41d9-a37f-2dab0ab06c54)

Why do I get CRITICAL warnings for battery voltage when it is clearly in the green zone? I believe the green is 12.4-15v and 13.20 is well within that range.

**Comment by griffths** [comment]
Jim,

 

I had the same question a while ago and some said that the check battery is a timer that goes off independent of the actual state of the battery and is simply a reminder to inspect the battery similar to the “inspect air filter” alert.

 

However, my experience was that the controller sensed the battery was failing because I would reset the alert only to have it set again a few days later.

 

Since replacing my battery, I have not received a check battery alert.

 

Regards,

 

Jay

 

From: Jim ***@***.***> 
Sent: Saturday, June 15, 2024 2:09 PM
To: jgyates/genmon ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jgyates/genmon] Critical Alert - Check Battery? (Discussion #1137)

 

Voltage.png (view on web) <https://github.com/jgyates/genmon/assets/16121417/d3611ab6-77e2-41d9-a37f-2dab0ab06c54> 

Why do I get CRITICAL warnings for battery voltage when it is clearly in the green zone? I believe the green is 12.4-15v and 13.20 is well within that range.

**Comment by jcleek** [comment]
Thanks Jay. It has been doing this since I replaced it so I don't know that replacing it again will matter.

**Comment by griffths** [comment]
Jim,

 

Did you reset the alert at the controller after replacing the battery?

**Comment by jcleek** [comment]
I did, and I have been resetting it about every other week for several months since it was replaced.

**Comment by jgyates** [comment]
The "Inspect Battery" should be the timed alarm, similar to inspect air filter alarm. The "check battery" is based on some checks that occur in the firmware related to charging (your milage may vary based on your model: air cooled, liquid cooled, evo1, ev2, nexus ,etc)

The text below mentions the "inspect battery" as a failure so I am not 100% sure. I also would not be suprised if this is dfferent for Nexus vs Evo and Liquid Cooled vs Air cooled.

Here is some info from the maintenance manual:
~
The charger operates at one of three battery charging voltage levels depending on ambient temperature.
• 13.5 VDC at High temperature
• 14.1 VDC at Normal temperature • 14.6 VDC at Low temperature
The battery charger is powered from a 120 VaC Load connection through a fuse (F3) in the transfer switch. this 120 VaC source must be connected to the generator in order to operate the charger.

During a utility failure, the charger will momentarily be turned off until the generator is connected to the Load. During normal operation, the battery charger supplies all the power to the Nexus controller.

the battery charger will begin its charge cycle when battery voltage drops below approximately 12.6V. the charger provides current directly to the battery dependant on temperature, and the battery is charged at the appropriate voltage level for 18 hours. at the end of the 18 hour charge period battery charge current is measured when the generator is off. if battery charge current at the end of the 18 hour charge time is greater than a pre-set level, or the battery open-circuit voltage is less than approximately 12.5V, an “inspect Battery” warning is raised. if the engine cranks during the 18 hour charge period, then the 18 hour charge timer is restarted.
at the end of the 18 hour charge period the charger does one of two things. if the temperature is less than approximately 40oF the battery is continuously charged at a voltage of 14.1V (i.e. the charge voltage is changed from 14.6V to 14.1V after 18 hours). if the temperature is above approximately 40oF then the charger will stop charging the battery after 18 hours.

the battery has a similar role as that found in an automobile application. it sits doing nothing until it either self-discharges below 12.6V or an engine crank occurs (i.e. such as occurs during the weekly exercise cycle). if either condition occurs the battery charge will begin its 18 hour charge cycle.

~

**Comment by jcleek** [comment]
@jgyates  Any way to add a button to reset an alarm? I have to physically go to the generator, unlock the top, then go through menus to reset the alarm.

**Comment by jgyates** [comment]
If your controller supports remote alarm reset then you will see a button for it on the Maintenance page. off the top of my head I think that is only the Evo Liquid Cooled models.

**Comment by jcleek** [comment]
:-\

I reject your reality and substitute my own. Seriously, thank you for the education. Maybe I do need to change the battery again.

**Reply by skipfire** [reply]
How old is the battery?  Not from install date, but from manufacture date?  Batteries that sit on the shelf for awhile can get a bit crappy.

**Reply by jcleek** [reply]
Thats a good question, and I don't know the answer. I will have to look next time I reset and see if it has a date on it.
